source/includes/api/_layer.md

```diff
@@ -350,185 +524,307 @@ curl -X GET https://api.resourcewatch.org/v1/layer?includes=user
+
+Field name              | Type           | Required             | Default value              | Description                                                                  
+----------------------- | -------------- | -------------------- |----------------------------| ---------------------------------------------------------------------------- 
+id                      | String         | Yes (autogenerated)  |                            | Unique Id of the layer. Auto generated on creation. Cannot be modified by users.    
+name                    | String         | Yes                  |                            | Name of the layer.              
+dataset                 | String         | Yes                  |                            | Id of the dataset to which the layer corresponds. Set on layer creation, cannot be modified.   
+slug                    | String         | Yes (autogenerated)  |                            | Slug of the layer. Auto generated on creation. Cannot be modified by users.  
+description             | String         | No                   |                            | User defined description of the layer.   
+application             | Array          | Yes                  |                            | Applications associated with this layer.                                   
+iso                     | Array          | No                   |                            | List of ISO3 codes of the countries that relate to the layer. If empty (or contains a single element: 'global') then the layer is a global layer.
+provider                | String         | No                   |                            | Layer provider. It typically identifies the source service for the data displayed in the layer.
+type                    | String         | No                   |                            | Layer type.
+userId                  | String         | Yes (autopopulated)  |                            | Id of the user who created the layer. Set automatically on creation. Cannot be modified by users.
+default                 | Boolean        | No                   | false                      | If the layer should be used as the dataset's default layer.
+protected               | Boolean        | Yes                  | false                      | If the layer is protected. A protected layer cannot be deleted.               
+published               | Boolean        | Yes                  | true                       | If the layer is published or not.                                               
+env                     | String         | Yes                  | production                 | Environment to which the layer belongs.
+applicationConfig       | Object         | No                   |                            | Open schema object meant to host application-specific data or behavior configuration.
```

Comment:
I would not recommend the phrase "Open schema" because it makes me think of Open Graph and schema.org, two competing web tagging standards. But I am open to the possibility that it's just me.

Reply:
I'm happy to change it to something else, but I can't put my finger on a better wording. Something like "open format" was the first thing that came to mind, but format, to me, sounds more like JSON vs XML/TXT/etc, which is not what we have here - the format is JSON, and it's not configurable. 
Any suggestions from your end?

Reply:
Maybe "schema-less object"?